NitroxClient/GameLogic/InitialSync/RemotePlayerInitialSyncProcessor.cs

```diff
@@ -62,15 +79,18 @@ public override IEnumerator Process(InitialPlayerSync packet, WaitScreen.ManualW
                         if (escapePod != null)
                         {
                             Log.Debug("Found escape pod for player. Will add him and update animation.");
-                            player.UpdateAnimation(AnimChangeType.UNDERWATER, AnimChangeState.OFF);
+                            swimming = false;
                         }
                     }
                     else
                     {
                         Log.Error("Could not spawn remote player into subroot/escape pod with id: " + playerData.SubRootId.Value);
                     }
                 }
-
+                if (!swimming)
```

Comment:
Does this also count for the player being above water on land or a base/lifeboat roof?

Reply:
I don't think we currently check that but as far as i see it, it could be easily added via
```C#
float oceanLevel = Ocean.main.GetOceanLevel();
swimming = position.y < oceanLevel;
```